src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStore.cs

```diff
@@ -56,12 +90,13 @@ public async Task<long> CreateInstanceIndexAsync(DicomDataset instance, Cancella
                     instance.GetSingleValueOrDefault<string>(DicomTag.PatientID),
                     instance.GetSingleValueOrDefault<string>(DicomTag.PatientName),
                     instance.GetSingleValueOrDefault<string>(DicomTag.ReferringPhysicianName),
-                    instance.GetStringDateAsDateTime(DicomTag.StudyDate),
+                    instance.GetStringDateAsDate(DicomTag.StudyDate),
                     instance.GetSingleValueOrDefault<string>(DicomTag.StudyDescription),
                     instance.GetSingleValueOrDefault<string>(DicomTag.AccessionNumber),
                     instance.GetSingleValueOrDefault<string>(DicomTag.Modality),
-                    instance.GetStringDateAsDateTime(DicomTag.PerformedProcedureStepStartDate),
-                    (byte)IndexStatus.Creating);
+                    instance.GetStringDateAsDate(DicomTag.PerformedProcedureStepStartDate),
+                    (byte)IndexStatus.Creating,
+                    parameters);
```

Comment:
How will the compat work? Basically I have V1 DB and V2 Code. You will have to check the schema version and call different SQL procedures.

Make sure same is achieved for other existing v1 SQL procedures who's signature is changing